parsl/data_provider/data_manager.py

```diff
@@ -47,13 +48,18 @@ class DataManager(ParslExecutor):
     to it, and DataFutures are returned.
     """
 
-    default_data_manager = None
-
     @classmethod
     def get_data_manager(cls, max_threads=None, executors=None):
```

Comment:
`max_threads` and `executors` are spurious arguments now - they're passed in to the constructor elsewhere now.

Reply:
good catch, thanks ben!